README.md

```diff
@@ -1 +1,45 @@
-# awesome-code-connector
\ No newline at end of file
+<div align="center">
+<a href="https://codeconnector.io/" target="_blank">
+	<img src="assets/ccLogos/codeconnectorLogo.png" alt="codeconnector">
+</a>
+	<br>
+	<h4>Open Source Learning Resources, compiled by the Dev Community for the Dev Community.</h4>
+	<br>
+	<br>
+	<br>
+</div>
+
+<div align="center">
+	<a href="codeconnector.md">About Code Connector</a>&nbsp;&nbsp;&nbsp;
+	<a href="contributionguidelines.md">Contribution Guidelines</a>&nbsp;&nbsp;&nbsp;
+</div>
+
+<br>
+
+<div align="center">
+<b> Connect with us </b>
+</div>
+
+<br>
+
+<p align="center">
+  <a href="https://codeconnector.io/" target="_blank"><img width="45" height="45" src="assets/ccLogos/ccLogo.jpg" alt="codeconnector"></a>&nbsp;&nbsp;&nbsp;
+	<a href="https://www.facebook.com/codeconnector" target="_blank"><img width="45" height="45" src="assets/socialMediaLogos/facebookLogo.png" alt="facebook"></a>&nbsp;&nbsp;&nbsp;
+	<a href="https://twitter.com/codeconnector_" target="_blank"><img width="45" height="45" src="assets/socialMediaLogos/twitterLogo.png" alt="twitter"></a>&nbsp;&nbsp;&nbsp;
+	<a href="https://www.youtube.com/channel/UCILxIdh3uyw64JjtdRmdtVg" target="_blank"><img width="45" height="45" src="assets/socialMediaLogos/youtubeLogo.png" alt="youtube"></a>&nbsp;&nbsp;&nbsp;
+	<a href="https://app.slack.com/client/T86NXDXQD/C86NXE3KP" target="_blank"><img width="45" height="45" src="assets/socialMediaLogos/slackLogo.png" alt="slack"></a>&nbsp;&nbsp;&nbsp;
+	<a href="https://www.linkedin.com/company/codeconnector/" target="_blank"><img width="45" height="45"src="assets/socialMediaLogos/linkedinLogo.png" alt="linkedin"></a>
+</p>
+
+<br>
+
+## Quick Navigation to Learning Resources
+
+- [General](./General)
```

Comment:
My preference is [kebab case](https://en.wiktionary.org/wiki/kebab_case) to avoid needing to think of wether the letters are capitalized or not to begin with. Do you have a strong preference to PascalCase?

Reply:
No strong preference, will change!